fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -48,7 +51,7 @@ def load_data(self, X, y, event=None, **kwargs):
                            names=[_SIGN, _EVENT, _GROUP_ID])
         self.index = signed.index
         self.default_objective_lambda_vec = None
-
+        self.multiplier = self.tags.groupby([_EVENT, _GROUP_ID]).mean()
```

Comment:
I'm struggling to understand what the multiplier is for, so I assume it's unclear to others as well (especially people who aren't as involved). Can we add a comment explaining what multiplier's purpose is?

Reply:
Sure. I think this is confusing. I'll write the documentation to explain this.

Reply:
Still struggling to understand what it looks like. Is it the mean of y-values (labels) of each group (as defined through event + group ID)? That would be 0/1. Overriding this for error rate would mean -1/1. Sorry, I feel like I'm missing something. I'll read your derivations again, but it needs additional description (at least for error rate) saying why we want -1/1 and what that does. It refers to Example 4 below, but example 4 is for equalized odds.

Reply:
I get where the confusion came from - it was kind of a hack the way I implemented it before. I have simplified the code now. Should be more readable and easy to understand. Also, I'm fixing the documentation to make it more transparent.

Reply:
Sounds good. I'll resolve this comment and will take a look at the current state :-)